Formula/vis.rb

```diff
@@ -26,6 +29,7 @@ class Vis < Formula
   def install
     # Make sure I point to the right version!
     lua = Formula["lua"]
+    ENV.append_to_cflags "-I#{lua.opt_include}/lua"
```

Comment:
Is there a reason that `pkg-config --cflags lua` isn't able to get correct include dir?
In theory, the test for `lua` runs `$CC $CFLAGS $CFLAGS_LUA` with the latter flags from `pkg-config`.

Reply:
Good question. I have no idea, tbh. It should already be in `CFLAGS_LUA`:
```
root@ed1288b20d86:/home/linuxbrew# pkg-config --cflags lua
-I/home/linuxbrew/.linuxbrew/include/lua
```
Would be useful to get `config.log` from the failed build to see what happened.

Reply:
Oh, wait. Their `configure` script looks like a handmade one, so it doesn't generate a `config.log`.

Reply:
Actually, I think I know the problem. The `pc` file we generate for `lua` in Homebrew/core is missing the `-ldl` flag. Making sure that flag is included by editing the `pc` file fixes the issue of the `configure` script not finding `lua`.

...it doesn't explain why my `append_to_cflags` solution worked though 🤷

Reply:
`append_to_cflags` probably worked due to the fall-through scenario if `pkg-config` fails or doesn't exist
```sh
		CFLAGS_LUA=""
		LDFLAGS_LUA="-l$liblua -lm -ldl"

		if $CC $CFLAGS $CFLAGS_LUA "$tmpc" \
			$LDFLAGS $LDFLAGS_LUA -o "$tmpo" >/dev/null 2>&1 ; then
```

Do other Linux distros include `-ldl`?
If not, then it may be an issue with upstream's `configure` script.

Reply:
I think some of them do. We do too, actually: https://github.com/Homebrew/linuxbrew-core/blob/24ee965a2eaabd7e87e77d19c55712cdee8aad94/Formula/lua.rb#L111

Reply:
Debian's `lua5.4.pc` file has:
```
Libs: -L${libdir} -l${lib_name}
Libs.private: -lm -ldl
```
While ArchLinux's file looks more like ours:
```
Libs: -L${libdir} -llua -lm
```